docs/install/install-node.rst

```diff
@@ -159,11 +159,8 @@ Paste them into a root console on the target node:
 .. task:: install_flocker centos-7
    :prompt: [root@centos]#
 
-Installing ``flocker-node`` will automatically install Docker, but the ``docker`` service may not have been enabled or started.
-To enable and start Docker, run the following commands in a root console:
-
-.. task:: enable_docker centos-7
-   :prompt: [root@centos]#
+Flocker's container management features depend on Docker.
+Make sure `a recent version of Docker is installed <https://docs.docker.com/docker/installation/>` and running.
```

Comment:
The minimum version that supports API v1.15 is 1.3.0, but FLOC-1059 says our minimum version should be 1.3.3, due to security problems.